lte/gateway/c/core/oai/test/spgw_task/test_spgw_procedures_with_injected_state.cpp

```diff
@@ -195,6 +195,9 @@ TEST_F(SPGWAppInjectedStateProcedureTest, TestDeleteSessionSuccess) {
 
   ASSERT_TRUE(eps_bearer_ctxt_p->paa.ipv4_address.s_addr == test_ue_ip);
 
+  ASSERT_TRUE(is_num_ue_contexts_valid(test_imsi64, name_of_ue_samples.size()));
+  ASSERT_TRUE(is_num_teids_valid(test_imsi64, 1));
```

Comment:
Can we rename as is_num_cp_teids_valid to indicate that we are validating control plane teids